apps/emqx/include/emqx_persistent_session_internals.hrl

```diff
@@ -68,7 +68,29 @@
     %% marked for deletion:
     unsubscribed = false :: boolean(),
     %% Reference to the subscription state:
-    sub_state_id :: emqx_persistent_session_ds_subs:subscription_state_id()
+    sub_state_id :: emqx_persistent_session_ds_subs:subscription_state_id(),
+    stream_owner :: emqx_persistent_session_ds:stream_owner()
```

Comment:
4> size(term_to_binary(owner_shared_sub)).
19
5> size(term_to_binary(owner_session)).
16
6> size(term_to_binary(0)).
3
7> size(term_to_binary(1)).
3

Reply:
...We store this things durably and transfer over RPC. Let's not do premature pessimization.